app/assets/stylesheets/application.css.scss

```diff
@@ -69,6 +69,18 @@ body {
 
 .application-body-bree{
   background-color: black;
+  background-image: url('bree.jpg');
+  background-size:     cover;
```

Comment:
Properties should be ordered background-color, background-image, background-position, background-repeat, background-size<br>Colon after property should be followed by one space